app/styleguide/wskComponentHandler.js

```diff
@@ -75,6 +75,11 @@ var componentHandler = (function() {
       var registeredClass = findRegisteredClass_(jsClass);
       if (registeredClass) {
         createdComponents_.push(new registeredClass.classConstructor(element));
+        // Call any callbacks the user has registered with this component type.
+        var length = registeredClass.callbacks.length;
+        for (var n = 0; n < length; n++) {
+          registeredClass.callbacks[n](element);
+        }
```

Comment:
Will not change: ForEach is 85% slower than a simple for loop. http://jsperf.com/for-vs-foreach/32